.travis.yml

```diff
@@ -111,12 +111,12 @@ script:
   - $COMPILER --version
   - ./autogen.sh
   - ./configure --enable-werror CC=$COMPILER
-  - make
-  - make -j9 check
-  - make -j9 distcheck
+  - make V=1
+  - make V=1 -j9 check || cat test/test-suite.log
```

Comment:
the `|| cat` breaks `set -e` enforcement and break travis build at that command. you may want to have:

```
  - make V=1 -j9 check || { cat test/test-suite.log; exit 1; }
```